clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -149,6 +153,11 @@ llvm::Constant *CodeGenModule::getBuiltinLibFunction(const FunctionDecl *FD,
         &getTarget().getLongDoubleFormat() == &llvm::APFloat::IEEEquad() &&
         F128Builtins.contains(BuiltinID))
       Name = F128Builtins[BuiltinID];
+    else if (getTriple().isPPC() && getTriple().isOSLinux() &&
+             &getTarget().getLongDoubleFormat() ==
+                 &llvm::APFloat::PPCDoubleDouble() &&
+             PPCDoubleDoubleBuiltins.contains(BuiltinID))
+      Name = PPCDoubleDoubleBuiltins[BuiltinID];
```

Comment:
Does adding this codepath actually change the computed name here?  The default codepath should just chop off the beginning of the string "__builtin_frexpl" to produce the same thing.

Reply:
I didn't realize that. Thanks for pointing it out.